internal/login/login.go

```diff
@@ -119,7 +119,10 @@ func listenForCode(state string) (code string, err error) {
 
 		// Use a separate thread so browser doesn't show a "No Connection" message
 		go func() {
-			server.Shutdown(context.TODO())
+			 err := server.Shutdown(context.TODO())
+			 if err != nil {
+				 fmt.Printf("Server shutdown error.")
+			 }
```

Comment:
I think I'd rather ignore this error. The reasoning is that if the user successfully authenticates, but the server fails to shut down, this doesn't actually pose an issue for the user. Adding the underscore should make `staticcheck` happy.
```suggestion
			 _ = server.Shutdown(context.TODO())
```